lib/fine.ex

```diff
@@ -8,11 +8,9 @@ defmodule Fine do
 
   @moduledoc readme_docs
 
-  @include_dir Path.expand("include")
-
   @doc """
   Returns the directory with Fine header files.
   """
   @spec include_dir() :: String.t()
-  def include_dir(), do: @include_dir
+  def include_dir(), do: Application.app_dir(:fine, "include")
```

Comment:
I was surprised that it works, but there is actually a symlink for `include/` in `_build/$MIX_ENV/lib/fine`:

```
fine main l _build/dev/lib/fine
drwxr-xr-x@ - jonatanklosko 19 Feb 09:20 consolidated
drwxr-xr-x@ - jonatanklosko 27 Jun 16:08 ebin
lrwxr-xr-x@ - jonatanklosko 18 Feb 20:00 include -> ../../../../include
```

I think this is a sheer coincidence that Mix adds the symlink, for what I imagine is Erlang `.hrl` files. It seems kinda wrong to rely on that behaviour for a different use case. @josevalim wdyt?

Reply:
Agreed. include has a well defined meaning for OTP apps.

Reply:
As a matter of fact, I'd say we do not want fine files in `_build`, because they are not necessary at runtime and therefore a wasted space in the release (they are small tho). I am actually going to change the dir name, so it doesn't get copied. Unfortunately, once I do that, the fix is not going to work anymore @kivikakk.